src/test/scala/cornichonTests/CheckoutExistProdTest.scala

```diff
@@ -1,66 +1,98 @@
 package cornichonTests
+import com.github.agourlay.cornichon.CornichonFeature
 import com.github.agourlay.cornichon.core.FeatureDef
+import com.github.agourlay.cornichon.http.{HttpRequest, RootExtractor}
+import com.github.agourlay.cornichon.resolver.JsonMapper
+import com.github.agourlay.cornichon.steps.regular.EffectStep
+import com.github.agourlay.cornichon.steps.wrapped.ScenarioResourceStep
 
-class CheckoutExistProdTest extends CheckoutTest with FeatureWithToken with FeatureWithProduct {
-
-  override lazy val baseUrl = apiUrl + s"/$projectKey"
+class CheckoutExistProdTest extends CornichonFeature with FeatureWithProduct {
 
   override def feature: FeatureDef = Feature("Checkout process for existing product") {
     Scenario("Add product | Create order | Cleanup") {
-      WithToken {
-        Then I createTaxCategory
-        Then I addProduct
-        Then I createCart
-        Then I addLineItem
-        Then I createOrderFromCart
-        Then I getCartById
-        Then I deleteOrderFromCart
-        Then I deleteCart
-        Then I deleteProduct
-        Then I deleteTaxCategory
-      }
-    }
-  }
+        Then assert addTaxCategoryStep
```

Comment:
A typical test is:
- Given XXX
- When XXXX
- Then XXX

for more background:
- https://martinfowler.com/bliki/GivenWhenThen.html
- https://www.agilealliance.org/glossary/gwt/#q=~(infinite~false~filters~(postType~(~'page~'post~'aa_book~'aa_event_session~'aa_experience_report~'aa_glossary~'aa_research_paper~'aa_video)~tags~(~'given*20when*20then))~searchTerm~'~sort~false~sortDirection~'asc~page~1)

Reply:
ok, thanks